src/store/utils.ts

```diff
@@ -25,3 +30,44 @@ export const convertForMulticall = <
 
   return contractCall;
 };
+
+type CreateWatcherSagaOptions = {
```

Comment:
shouldn't utils have their own types?

Reply:
Or maybe we can put it in `src/store/test.ts`

Reply:
> Or maybe we can put it in `src/store/test.ts`

You mean `src/store/types.ts`, right? If so, I totally agree